Cargo.toml

```diff
@@ -40,7 +40,7 @@ dpdk-sys = { path = "./dpdk-sys" }
 dpdk-sysroot-helper = { path = "./dpdk-sysroot-helper" }
 dplane-rpc = { git = "https://github.com/githedgehog/dplane-rpc.git", version = "1.0.0" }
 errno = { path = "./errno", package = "dataplane-errno" }
-gateway_config = { git = "https://github.com/githedgehog/gateway-proto", version = "0.1.0" }
+gateway_config = { git = "https://github.com/githedgehog/gateway-proto", branch ="pr/smatov/ospf-initial" }
```

Comment:
Uh

Reply:
Can we replace this branch name with an existing tag or commit SHA from main branch, until we switch it to v0.2.0 in a later commit please? This branch will eventually move or disappear, and if we try to bisect on this range of commits we won't be able to compile anymore.

Reply:
This is replaced by @sergeymatov in a branch on top of this one.

Reply:
Yes, I still believe it would be a good idea to replace it with a more permanent commit SHA than the name of a branch that will disappear soon. But then we're lifting the restriction on individual commits being able to build so I guess it's up to you :shrug:

Reply:
I would not bother fixing this now.